bench/map.carp

```diff
@@ -4,21 +4,21 @@
 (defn insert []
   (let [m {}]
     (for [i 0 100]
-      (set! m (Map.put &m &i &1)))))
+      (Map.put! &m &i &1))))
```

Comment:
It seems like these are unrelated to the NixOS change, or am I mistaken?

Reply:
I changed those so the Nix build can execute the tests correctly.

Reply:
Ah, interesting! What about Nix made it necessary?

Reply:
The nix recipe runs the test suite unless told otherwise. If it fails, the whole installation fails.

Reply:
Oh, so you’re saying that the old implementation was actually broken?

Reply:
Yes, it simply didn't compile.

Reply:
Oh wow, that’s weird. For me `test/map.carp` works fine, or was it triggered by a different part of the test suite? For me `run_carp_tests.sh` seems to run just fine on the latest version in `master`.

Reply:
It's `bench/map.carp`, so I guess It wasn't due to the test suite but due to me manually testing the `benchmarks.sh` script.

Reply:
Ah, perfect, thank you! For some reason I was thinking we were looking at `core/Map.carp` and was confused how it was possible that it didn’t compile. Sorry about that!